spec/openapi.yml

```diff
@@ -478,6 +478,15 @@ paths:
       tags: [sandboxes]
       security:
         - ApiKeyAuth: []
+      parameters:
```

Comment:
Do we want just simple key-value pairs or do we want to support something more advanced? If so, I would probably change this to JSON